### ISSUE TITLE
Bug: the default value of "space-in-parens" is "always" (expected "never")

### ISSUE THREAD

**Issue by vyshkant** [buzinas/tslint-eslint-rules#305]
### TSLint version:
```bash
$ tslint --version
5.9.1
```

### TypeScript version:
```bash
$ tsc --version
Version 2.7.2
```

### tslint.json configuration:
```json
{
  "extends": [
    "tslint-eslint-rules"
  ],
  "rules": {
    "space-in-parens": [true],
  }
}
```

### TypeScript code being linted:
```typescript
if (variable <= 0) {
    throw new Error('Message');
}
```

### Actual behavior:
```bash
ERROR: /path/to/my/file.tsx[...]: there must be a space inside this paren.
ERROR: /path/to/my/file.tsx[...]: there must be a space inside this paren.
ERROR: /path/to/my/file.tsx[...]: there must be a space inside this paren.
ERROR: /path/to/my/file.tsx[...]: there must be a space inside this paren.
```
(the positions of expected space are near these four parens)

### Expected behavior:
No errors, because expected default value of "space-in-parens" is "never".

**Comment by buzinas** [comment]
Hey @vyshkant, thanks for spotting this!

Would you mind to go ahead and create a PR changing this behavior and also adding a test that ensures the default value is `never` when no parameters are passed?

**Comment by vyshkant** [comment]
Sure, I'll gladly write a PR! If it can wait a few days :smile:
